src/scenarios/shared/const.py

```diff
@@ -14,3 +16,14 @@
 PUBDIR = 'pub'
 APPDIR = 'app'
 TRACEDIR = 'traces'
+TMPDIR = 'tmp'
+
+DOTNET = 'dotnet'
+PYTHON = 'py'
+
+BUILD_CLEAN = 'Clean Build'
+BUILD_NO_CHANGES = 'Build No Changes'
```

Comment:
Do we really need these to be defined in const? We seem to only use them in one place.

Reply:
u r right. no good reason to make it constant for now